app/src/main-process/menu/build-default-menu.ts

```diff
@@ -288,6 +288,12 @@ export function buildDefaultMenu(
         accelerator: 'CmdOrCtrl+Shift+U',
         click: emit('update-branch'),
       },
+      {
+        label: __DARWIN__ ? 'Compare to Branch' : '&Compare to branch',
```

Comment:
This is the same problem as was uncovered in #4618 - both `&Compare to branch` and `&Compare on GitHub` cannot `&` the same letter. I'd suggest moving the latter to `Compare on &GitHub`

Reply:
What is that ampersand represent in this case? And what is the level of uniqueness it must have? I see we use `&G...` in multiple places. Right now, I'm assuming sub-menus have their own scope, but I could not verify using the [docs](https://electronjs.org/docs/api/menu). Also, is there a method for picking which character to put the ampersand next to or is it arbitrary?

Reply:
@iAmWillShepherd i found this in some documentation
> To create an access key for a menu item, precede any character in the item's text string with an ampersand. For example, the text string "&Move" causes the system to underline the letter "M".

https://msdn.microsoft.com/en-us/library/windows/desktop/ms647553(v=vs.85).aspx

i also found a helpful tip (For windows)
> Make sure all the access keys on a menu are unique by right-clicking your menu and choosing Check Mnemonics from the shortcut menu.

https://msdn.microsoft.com/en-us/library/9sewd7e1.aspx

i found some more best practices here: https://webaim.org/techniques/keyboard/accesskey
that indicates that the access key should be the character you show for the shortcut but nothing else that helped me understand which character

Reply:
@nerdneha It looks like that WebAIM link is for the HTML `accesskey` attribute, which is different from the shortcuts used here.

Reply:
> What is that ampersand represent in this case? 

In Windows applications, when you press <kbd>ALT</kbd> the application menu becomes focused, with the access keys underlined:

<img width="406" src="https://user-images.githubusercontent.com/359239/40207213-c429c212-5a76-11e8-9342-8e60a9b3d896.png">

This shows that you can open a given menu by pressing the relevant access key.

Submenus can also have access keys:

<img width="264"  src="https://user-images.githubusercontent.com/359239/40207241-e00031ce-5a76-11e8-85cb-b3b254996b73.png">

> And what is the level of uniqueness it must have?

The uniqueness does not need to be global, just within the given menu or submenu. In fact we have an example here where the top-level `Branch` menu item and the inner `New branch...` menu entries both use `B` (it's not case-sensitive) as a shortcut. This is fine, and if you do <kbd>ALT</kbd> + <kbd>B</kbd> + <kbd>B</kbd> you'll quickly open the new branch dialog.

> Also, is there a method for picking which character to put the ampersand next to or is it arbitrary?

I'm not sure of the rules here beyond my gut instincts, but essentially you want something that stands out - either the first letter of a sentence-cased word, or a scope-unique letter near the start of the menu label.